config/jobs/kubernetes/kops/kubernetes-presubmits.yaml

```diff
@@ -36,6 +36,7 @@ presubmits:
         - --aws-cluster-domain=test-cncf-aws.k8s.io
         - --build=bazel
         - --cluster=
+        - --env=KOPS_ARCH=amd64
```

Comment:
@hakman 
I assume this will take effect once Kops cut a new 1.19 release?

Reply:
Should work now if you re-run the job.